resources/lib/method.py

```diff
@@ -217,14 +239,14 @@ def _get_preselected_items(string):
             'params': (
                 ('heading', str, ''),
                 ('list', _split_items, ''),
-                ('autoclose', int, 0), ('preselect', _get_preselected_items, -1), ('useDetails', boolean, False), )
+                ('autoclose', int, 0), ('preselect', _get_preselected_item, -1), ('useDetails', boolean, False), )
         },
         'multiselect': {
-            'func': dialog.select,
+            'func': dialog.multiselect,
             'params': (
                 ('heading', str, ''),
-                ('list', _split_items, ''),
-                ('autoclose', int, 0), ('preselect', _get_preselected_items, -1), ('useDetails', boolean, False), )
+                ('options', _split_items, ''),
```

Comment:
Not a problem but just a note at how annoying that the kwarg name is not the same for multiselect as select!